assets/javascripts/discourse/controllers/user-activity-my-reactions.js.es6

```diff
@@ -0,0 +1,42 @@
+import Controller, { inject as controller } from "@ember/controller";
+import { observes } from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import CustomReaction from "../models/discourse-reactions-custom-reaction";
+
+export default Controller.extend({
+  canLoadMore: true,
+  loading: false,
+  application: controller(),
+
+  @action
+  loadMore() {
+    if (!this.canLoadMore || this.loading || !this.reactionsUrl) {
+      return;
+    }
+
+    this.set("loading", true);
+    const posts = this.model;
+
+    const beforeReactionUserId = posts.length
+      ? posts[posts.length - 1].get("id")
```

Comment:
what is get achieving here ?

Reply:
Oooh actually its a mistake, i had to change the variable name `posts` to `reactions`